tests/integration/runner

```diff
@@ -276,6 +289,14 @@ if __name__ == "__main__":
         help="Set network driver for runnner container (defaults to `host`)",
     )
 
+    parser.add_argument(
+        "--memory",
+        # Default is host memory - 2GiB
+        default=physical_memory() - 2 * 1024**3,
```

Comment:
would be nice to validate the result value I think, check that it is >0 or >PHYSICAL_MEMORY/2

Reply:
It's legal `docker run --memory` argument. I've checked

> -m , --memory=number[*S]
>   Memory limit; S is an optional suffix which can be one of b, k, m, or g.
>
> Allows  you  to  constrain the memory available to a container. If the host supports swap memory, then the -m memory setting can be larger than physical RAM. If a limit of 0 is specified (not using -m), the container's memory is not limited. The actual limit may be rounded up to a multiple of the operating system's page size (the value would be very large, that's millions of trillions).